operator/pkg/helm/helm.go

```diff
@@ -148,6 +149,18 @@ func renderChart(namespace, values string, chrt *chart.Chart) (string, error) {
 			return "", err
 		}
 	}
+	for _, crdFile := range crdFiles {
+		f := string(crdFile.File.Data)
+		// add yaml separator if the rendered file doesn't have one at the end
```

Comment:
Just add the YAML separator, who cares?

Reply:
Not sure - just copied the code above :slightly_smiling_face: